presto-main/src/main/java/com/facebook/presto/SystemSessionProperties.java

```diff
@@ -180,6 +181,11 @@ public SystemSessionProperties(
                         "Use grouped execution for aggregation when possible",
                         featuresConfig.isGroupedExecutionForAggregationEnabled(),
                         false),
+                booleanProperty(
+                        DYNAMIC_SCHEDULE_FOR_GROUPED_EXECUTION,
+                        "Use dynamic schedule for grouped execution when possible",
+                        false,
```

Comment:
Should we have a configuration property for this, so it can globally be enabled or disabled?

Reply:
@dain : I was originally thinking it's a session only property since we want to remove it. If we have a configuration property, does it mean we need to first mark it as deprecated and wait for a few releases ? 

Add configuration property for now :)

Reply:
Yeah, I see grouped execution for aggregation and lifespan per task all have configuration properties. I guess we should also have configuration property for this one as well